Expensify/App
# Fix notification onlick doesn't focus browser or tab in Firefox

### Details
 
In Firefox notification disappears and the browser won't open after clicking on it because `window.parent.focus();` or `window.focus()`, doesn't work by default in Firefox. The default event is to focus the tab, but we are preventing it, so we must disable `preventDefault` for Firefox.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7325

### Tests


1. Enable notification for Expensify in firefox
2. In Firefox, open Expensify tab and other tab and focus to other tab.
3. Send a message from other device to 
4. Click the notification of the new message
5. Verify Expensify tab is focused/displayed and active room is changed  to the source of notification room. 
### PR Review Checklist

#### Contributor (PR Author) Checklist
- [ ] I followed proper code patterns (see [Reviewing the code](../PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I followed style guidelines (in [`Styling.md`](../STYLING.md)) for all style edits I made
- [ ] I followed the guidelines as stated in the [Review Guidelines](../PR_REVIEW_GUIDELINES.md)

#### PR Reviewer Checklist
- [ ] I verified proper code patterns were followed (see [Reviewing the code](../PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified [style guidelines](../STYLING.md) were followed
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](../PR_REVIEW_GUIDELINES.md)

### QA Steps



1. Enable notification for Expensify in firefox
2. In Firefox, open Expensify tab and other tab and focus to other tab.
3. Send a message from other device to 
4. Click the notification of the new message
5. Verify Expensify tab is focused/displayed and active room is changed  to the source of notification room.

### Tested On

- [x] Web Firefox

### Screenshots


#### Web


https://user-images.githubusercontent.com/1313124/155444602-876a4d45-8c59-45fc-849f-fb326798168a.mp4


#### Mobile Web

`Not affected`

#### Desktop


https://user-images.githubusercontent.com/1313124/156903519-a00291f2-52e2-40dc-9545-9d4015dbd424.mp4


#### iOS

`Not affected`

#### Android

`Not affected`
